spec/workflow/intrinsic_function_spec.rb

```diff
@@ -450,6 +450,13 @@
       end
     end
 
+    describe "States.JsonToString" do
+      it "fetches values from the input" do
+        result = described_class.value("States.JsonToString($.input)", {"context" => {"baz" => "qux"}}, {"input" => {"foo" => "bar"}})
+        expect(result).to eq("{\"foo\":\"bar\"}")
+      end
```

Comment:
Please include a test for the various failure cases (wrong type, too few params, too many params).